apicast/src/oauth/keycloak.lua

```diff
@@ -237,7 +237,7 @@ function _M:authorize(service, client)
   end
 
   local url = resty_url.join(self.config.authorize_url, ngx.var.is_args, ngx.var.args)
-  local res = http_client.get(url)
+  local res = ngx.redirect(url)
```

Comment:
ngx.redirect does not return anything and exits handling the request like ngx.exit